e2e-tests/playwright/utils/UIhelper.ts

```diff
@@ -90,7 +90,21 @@ export class UIhelper {
   }
 
   async openSidebar(navBarText: string) {
-    await this.page.locator(`nav a`).filter({ hasText: navBarText }).click();
+    await this.page.click(`nav a:has-text("${navBarText}")`);
+  }
+
+  async verifyTableHeaders(expectedHeaders: string[]) {
+    for (const header of expectedHeaders) {
+      const headerLocator = this.page.locator(
+        `${UIhelperPO.MuiTable} thead th span div >> text="${header}"`,
+      );
+      const count = await headerLocator.count();
+      if (count === 0) {
+        console.log(`Header "${header}" not found.`);
```

Comment:
it looks just console log, can we add assertions here?

Reply:
@subhashkhileri fixed